### ISSUE TITLE
Add support for villager spawning control

### ISSUE THREAD

**Issue by PureLove-X** [Multiverse/Multiverse-Core#1845]
Villagers are able to spawn with Monsters and Animals off, Is there any way to fix this. '

**Comment by Jadiction** [comment]
I have a world where monsters and animals are off but villagers do not spawn. I do, however sometimes see wandering villagers but they are new entities and I am using an updated Minecraft version. Noting this for later.

**Comment by seema84** [comment]
can confirm in 1.16.3 and Build #812

**Comment by benwoo1110** [comment]
Yep, bcu it’s not a feature at all to block villager spawning. Probably not going to add anytime soon as you can use you well build plugins like worldguard.

**Comment by seema84** [comment]
i think with a little attention here in the issue it will also be added

**Comment by benwoo1110** [comment]
mhmm, we are always opened to PR ;)

**Comment by seema84** [comment]
if I could, I would implement it directly :)

**Comment by CDWimmer** [comment]
Please god let me turn off wandering traders
I have a stick that makes lighting specifically for when I see them wandering my creative world

**Comment by benwoo1110** [comment]
This has been implemented with mv5 with the revamped entity spawn config
